simulators/ethereum/eest/execute-blobs/Dockerfile

```
@@ -0,0 +1,24 @@
+# Builds and runs the EEST (execution-spec-tests) consume engine simulator
+FROM ghcr.io/astral-sh/uv:python3.10-bookworm-slim
+
+## Default git-ref/fork
+ARG branch=main
+ENV GIT_REF=${branch} 
+ARG fork=Osaka
+ENV FORK=${fork}
+
+## Clone and install EEST
+RUN apt-get update && apt-get install -y git
+
+# Allow the user to specify a branch or commit to checkout
+RUN git init execution-spec-tests && \
+    cd execution-spec-tests && \
+    git remote add origin https://github.com/ethereum/execution-spec-tests.git && \
+    git fetch --depth 1 origin $GIT_REF && \
+    git checkout FETCH_HEAD;
+
+WORKDIR /execution-spec-tests
+RUN uv sync
+
+## Define `consume engine` entry point using the local fixtures
+ENTRYPOINT uv run execute hive --fork=$FORK -v -m get_blobs
```

```suggestion
ENTRYPOINT uv run execute hive --fork=$FORK -v -m get_blobs
```
About the `-m get_blobs`...

Here's just one other small bit of relevant info... and that's that we only parametrize a simulator with the format if it's supported by the particular simulator (`_supported_fixture_formats` is defined in the simulator's local `conftest.py`):

```python
def pytest_generate_tests(metafunc):
    """
    Generate test cases for every test fixture in all the JSON fixture files
    within the specified fixtures directory, or read from stdin if the directory is 'stdin'.
    """
    if "cache" in sys.argv:
        return

    test_cases = metafunc.config.test_cases
    param_list = []
    for test_case in test_cases:
        if test_case.format.format_name not in metafunc.config._supported_fixture_formats:
            continue
```
https://github.com/ethereum/execution-spec-tests/blob/020f728eefedc75a68e6fba6dbc65322e1d9f490/src/pytest_plugins/consume/consume.py#L411-L423

This avoids having to add these command-line args, which is beneficial if we want to allow the user to specify other markers (as in #1248).